packages/sdk/src/bot/implementation.ts

```diff
@@ -95,17 +88,13 @@ export class Bot {
     this.definition = def
   }
 
-  public register = (_handler: RegisterHandler): void => {}
-  public unregister = (_handler: UnregisterHandler): void => {}
-  public message = (_type: string, handler: MessageHandler): void => {
+  public message = (handler: MessageHandler): void => {
     this._state.messageHandlers.push(handler)
   }
-  public conversation = (_type: string, _handler: MessageHandler): void => {}
```

Comment:
I don't know what was the intention with the `conversation` and `user` methods, but I'm confident that we can provide primitives that allow the desired level of abstraction while using the "raw" callbacks like `message()` and `event()` to remain backward compatible